src/components/layout/header.tsx

```
@@ -2,18 +2,178 @@ import { NavLink, useNavigate } from "react-router-dom";
-  `rounded-full px-4 py-2 text-sm font-medium transition ${
-    isActive
-      ? "bg-slate-900 text-white"
-      : "text-slate-600 hover:bg-slate-100 hover:text-slate-900"
-  }`;
+const SearchIcon = () => (
+  <svg
+    className="h-5 w-5 text-gray-400"
+    fill="none"
+    stroke="currentColor"
+    viewBox="0 0 24 24"
+  >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)